roles/common/templates/bash_prompt.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+PROMPT_COMMAND="export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h[{{ environment_name }}]\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '"
```

Comment:
Actually, the failure here looks like shellcheck being overzealous, not ansible.